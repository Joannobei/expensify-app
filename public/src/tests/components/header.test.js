import React from 'react';
import { shallow, configure } from 'enzyme';
import Header from '../../components/Header';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render Header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    // expect(wrapper.find('h1').length).toBe(1);
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});
