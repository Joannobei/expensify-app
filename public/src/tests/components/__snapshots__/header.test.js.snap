// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home Page
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Create Page
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/help"
  >
    Help Page
  </[object Object]>
</header>
`;
